src/pages/projects.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
```

Comment:
@Alphascythian Let's call this file `project.js` instead of `projects.js` because this is the individual page of a singular project